### PR TITLE
fix,feat: Ability to work with Oracle`s tablespace feature without errors

### LINKED ISSUE #7032: Correct migration generate with Oracle`s tablespace
## Feature Description

Ability to work with Oracle`s Tablespace

### The Solution

Add code to solve it ( I have it in my production environment)

### Relevant Database Driver(s)

Oracle

- [ ] `aurora-data-api`
- [ ] `aurora-data-api-pg`
- [ ] `better-sqlite3`
- [ ] `cockroachdb`
- [ ] `cordova`
- [ ] `expo`
- [ ] `mongodb`
- [ ] `mysql`
- [ ] `nativescript`
- [X ] `oracle`
- [ ] `postgres`
- [ ] `react-native`
- [ ] `sap`
- [ ] `sqlite`
- [ ] `sqlite-abstract`
- [ ] `sqljs`
- [ ] `sqlserver`

### Are you willing to resolve this issue by submitting a Pull Request?

- [ ] Yes, I have the time, and I know how to start.
- [X ] Yes, I have the time, but I don't know how to start. I would need guidance.
- [ ] No, I don't have the time, although I believe I could do it if I had the time...
- [ ] No, I don't have the time and I wouldn't even know how to start.

### PR DESCRIPTION
**Pull request by osirisbest**
### Description of change

Ability to work with Oracle`s tablespace feature in extra section without errors ( like now).

Correct migration generation with Oracle`s tablespace #7032
Sample =>
"extra": {
    "connectString": "*****teq.com:1521/orcl",
    "tablespace": "SOLAR"
  }
### Pull-Request Checklist

- [X] Code is up-to-date with the `master` branch
- [X] `npm run lint` passes with this change
- [X] `npm run test` passes with this change
- [X] This pull request links relevant issues as `Fixes #7032`
- [ ] There are new or updated unit tests validating the change
- [ ] Documentation has been updated to reflect this change
- [ ] The new commits follow conventions explained in [CONTRIBUTING.md](https://github.com/typeorm/typeorm/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/driver/sqljs/SqljsQueryRunner.ts [hunk 1]
```diff
@@ -8,12 +8,12 @@ import {QueryFailedError} from "../../error/QueryFailedError";
  * Runs queries on a single sqlite database connection.
  */
 export class SqljsQueryRunner extends AbstractSqliteQueryRunner {
-    
+
```
Comment (nebkat) [inline src/driver/sqljs/SqljsQueryRunner.ts:11]: Probably best to avoid these changes in unrelated files at least

Comment (osirisbest) [reply src/driver/sqljs/SqljsQueryRunner.ts:11]: OK, may be i need close this proposal and open new? It`s my first proposal and i wanted to fix some style errors too.

Comment (nebkat) [reply src/driver/sqljs/SqljsQueryRunner.ts:11]: In any project it is generally better to keep changes separate to make it easier to find bugs, explanations for changes, etc. You can fix style errors that are right beside the code that you are changing but you should avoid doing it elsewhere. Best way to do this is to create a new branch on your repo specifically for this change, and only push the relevant commits there. Then when the pull request is being reviewed, the reviewers can focus only on the main problem and it can get merged quicker.

**Comment by AlexMesser** [comment]
If you are still interested in these changes, can you please submit a new PR with:

* actual implementation of this feature working. I don't see changes in SQL generation, correct me if I miss something
* without style changes, so it can be easier to review.

Thank you!
